apps/meeting/src/app.tsx

```diff
@@ -1,27 +1,27 @@
 // Copyright 2020-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: MIT-0
 
-import React, { FC } from 'react';
-import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
-import { ThemeProvider } from 'styled-components';
+import React, { FC } from "react";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+import { ThemeProvider } from "styled-components";
 import {
   lightTheme,
   MeetingProvider,
   NotificationProvider,
   darkTheme,
   GlobalStyles,
   VoiceFocusProvider,
-} from 'amazon-chime-sdk-component-library-react';
+} from "amazon-chime-sdk-component-library-react";
 
-import { AppStateProvider, useAppState } from './providers/AppStateProvider';
-import ErrorProvider from './providers/ErrorProvider';
-import routes from './constants/routes';
-import { NavigationProvider } from './providers/NavigationProvider';
-import { Meeting, Home, DeviceSetup } from './views';
-import Notifications from './containers/Notifications';
-import NoMeetingRedirect from './containers/NoMeetingRedirect';
-import MeetingEventObserver from './containers/MeetingEventObserver';
-import meetingConfig from './meetingConfig';
+import { AppStateProvider, useAppState } from "./providers/AppStateProvider";
+import ErrorProvider from "./providers/ErrorProvider";
+import routes from "./constants/routes";
+import { NavigationProvider } from "./providers/NavigationProvider";
+import { Meeting, Home, DeviceSetup } from "./views";
+import Notifications from "./containers/Notifications";
+import NoMeetingRedirect from "./containers/NoMeetingRedirect";
+import MeetingEventObserver from "./containers/MeetingEventObserver";
+import meetingConfig from "./meetingConfig";
```

Comment:
Any reason for the change on this double quotes? In JS SDK, other demos and React library imports have been single quote so can we keep these as single quotes?